data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,425 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Unfettered: Hunger!"
+	landing
+	invisible
+	repeat
+	to offer
+		not "event: wanderers: unfettered invasion starts"
+		# to not make it restart if it's already going and is triggered by the Hai Escort mission and only make it last longer
+		not "event: unfettered: need for food"
+		or
+			has "unfettered: hunger"
+			and
+				has "Unfettered: Jump Drive 2: offered"
+				# 1 chance in 500
+				random < 1
+				random < 20
+	on offer
+		event "unfettered: need for food"
+	on enter
+		system
+			government "Hai"
+		dialog `The local Hai authorities hail you. "Captain, We need your help! Our brethren up north are starving! The situation is even more dire than usual, and requires your immediate assistance. You will of course be well rewarded, as usual. We added a new job to respond to this crisis."`
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			branch "not first time"
+				"Unfettered: Hunger!: offered" > 1
+			`Surprisingly, the local Unfettered authorities hail you. In fact, they seem to be relaying a message to all ships in range, both in human and Hai... Their situation must be desperate.`
+				goto message
+			label "not first time"
+			`The local Unfettered authorities hail you. In fact, they are hailing all ships, in human and hai. This is not the first time they do this, and it probably means that, like last time, another famine is happening. If that were to be the case, they most likely did not manage their food issue in a permanent manner.`
+			label message
+			`	"Attention, to all that are sympathetic to our cause, or simply want to help a people survive, we are in great need of food. As such, we have negotiated an increase of tributes from the Hai, and they have offered some food supplies, as well as specialists to help us stabilize our food production. That is what we need all of you for, captains, all over our worlds parents are starving because they give what little they have to their children, to their future.`
+			`	You can almost feel them laying low their pride to say the next part, "We are not used to nor comfortable with begging, but please, help us! Spread the word, bring food, we will reward you handsomely as ever! Do it for the money if you must, or do it because you care."`
+	on daily
+		"unfettered: hunger stops" += random
+	to fail
+		or
+			# lasts approximately 60 days if the player does nothing
+			"unfettered: hunger stops" > 3000
+			has "event: wanderers: unfettered invasion starts"
+	on fail
+		event "unfettered: need for food stops"
+		dialog "You receive a message notifying you that with the extra food brought, the famine amongst the Unfettered has recessed, hopefully permanently."
```

Comment:
I feel like this would be a good place for it to say, "That’s what they said last time."
Not sure how to work that in, but…

Reply:
well this is the same text for the first time and after it so idk if I can add it

Reply:
I suppose branches don’t work for dialogs… can you use `to display`?

Reply:
not yet at least

Reply:
If this can’t be done, it can probably be resolved.